Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -3785,6 +3785,26 @@ def batch(iterable, batch_size=1):
         current_batch = not_batched[:batch_size]
         not_batched = not_batched[batch_size:]
 
+def safe_get(dict_object, *keys, key_return_value = None):
+    """ Recursive safe get query, If keys found return value othewisw return None
+
+    Args:
+        key_return_value: Value to return when no key availble
+        dict_object: dictionary to query.
+        *keys: keys for recursive get.
+
+    Returns:
+        :return: Value from recursive get.
+        :rtype: ``obj``
```

Comment:
fix docs format (see the other functions) - this is used for the script helper so docs must follow to format